storage/api/Storage.Samples.Tests/StorageFixture.cs

```diff
@@ -222,4 +236,19 @@ public void CollectArchivedFiles(string bucketName, string objectName, long? ver
         }
         versions.Add(version.Value);
     }
+
+    public Topic createTopic(string topicId)
+    {
+        PublisherServiceApiClient publisherClient = PublisherServiceApiClient.Create();
+        TopicName topicName = new TopicName(ProjectId, topicId);
+        var topic = publisherClient.CreateTopic(topicName);
+
+        var policy = new Google.Cloud.Iam.V1.Policy();
+        policy.AddRoleMember("roles/pubsub.publisher", "allUsers");
+        publisherClient.SetIamPolicy(topicName, policy);
```

Comment:
Can't you create the topic with the policy set, to try an avoid two backend calls?

Reply:
I could'nt find another way to do this which incorporates both calls